modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm

```diff
@@ -0,0 +1,132 @@
+/*
+*	MELEE
+*/
+
+/mob/living/basic/abductor
+	name = "abductor scientist"
+	desc = "From the depths of space."
+	icon = 'modular_skyrat/modules/awaymissions_skyrat/icons/abductors.dmi'
+	icon_state = "abductor_scientist"
+	icon_living = "abductor_scientist"
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	sentience_type = SENTIENCE_HUMANOID
+	faction = list(ROLE_ABDUCTOR)
+
+	maxHealth = 120
+	health = 120
+	unsuitable_atmos_damage = 7.5
+	basic_mob_flags = DEL_ON_DEATH
+
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	attack_sound = 'sound/weapons/punch1.ogg'
+
+	combat_mode = TRUE
+	status_flags = CANPUSH
+	speed = 2
+
+	ai_controller = /datum/ai_controller/basic_controller/abductor
+
+	/// What this mob drops on death
+	var/loot = list(/obj/effect/gibspawner/generic, /obj/effect/spawner/random/astrum/sci_loot)
```

Comment:
typecast to a list and (maybe) static, will want to check how it works w/ subtypes just to be safe